src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -1074,22 +1125,25 @@ describe('CdkVirtualScrollViewport', () => {
         .toBe(0);
     }));
 
-    it('should set content offset to bottom of content', fakeAsync(() => {
+    // TODO: verify
+    it('should set content offset to bottom of content', fakeAsync(async () => {
```

Comment:
Verify: replacing `detectChanges()` with `markForCheck(); detectChanges()` wasn't enough, had to switch to `autoDetectChanges()` / `whenStable()`

Reply:
It seems like the flush is needed for now, but I can try to do a pass to cleanup the fakeAsync usages at some point